k8s-opensearch/_variables.tf

```diff
@@ -0,0 +1,42 @@
+variable "md_metadata" {
+  type        = any
+  description = "md_metadata object"
+}
+
+// TODO this is only used in configuring my local dev provider. remove before merging.
+variable "kubernetes_cluster" {
+  type        = any
+  description = "A kubernetes_cluster artifact"
+}
+
+variable "release" {
+  description = "Release name"
+  type        = string
+  default     = "opensearch"
```

Comment:
Let's remove the default on this and `namespace`.  If this were a "normal" terraform module, I would agree defaults are nice. In our case these modules are just the DRY up the terraform, and we definitely want the module user to specify `release` and `namespace`.